src/test/java/edu/iis/powp/gui/TestPlotSoftPatterns.java

```diff
@@ -89,11 +103,11 @@ public void run()
                 ApplicationWithDrawer.configureApplication();
                 Context context = Application.getComponent(Context.class);
                 
-                setupDefaultDrawerVisibilityManagement(context);
-                
+                //setupDefaultDrawerVisibilityManagement(context);
```

Comment:
Jeżeli w kodzie nie jest potrzebne wywołanie metody to usuwamy całą linię, poprawia to czytelność kodu.